assets/data/cards_pt-br.json

```
@@ -98,6 +98,21 @@
       "description": "HTML é uma linguagem de marcação utilizada na construção de páginas na web. Documentos HTML podem ser interpretados por navegadores.",
       "tags": ["Front-end"]
     },
+    {
+      "title": "DOM (Document Object Model)",
+      "description": "Uma representação da estrutura de árvore dos elementos HTML em uma página, que permite a manipulação dinâmica do conteúdo. Imagine isso como um mapa da página que o JavaScript usa para entender e alterar o conteúdo. É como um 'modelo' da página.",
+      "tags": ["Front-end"]
+    },
+    {
+      "title": "Responsividade",
+      "description": "A capacidade de um site ou aplicativo se adaptar a diferentes tamanhos de tela e dispositivos, como smartphones, tablets e desktops.",
+      "tags": ["Front-end"]
+    },
+    {
+      "title": "SASS/SCSS",
+      "description": "Pré-processadores CSS que permitem a criação de estilos mais complexos e reutilizáveis. São formas mais avançadas de escrever estilos (cores, fontes) para sites.",
```

Eu acho que aqui poderia ser um pouquinho mais didático, que tal:

```suggestion
      "description": "Pré-processadores CSS são como ferramentas especiais que facilitam a criação de estilos para sites. Eles tornam possível fazer estilos mais complicados e que podem ser usados de novo em diferentes partes do site. Pense neles como formas mais espertas de escolher cores e fontes para tornar um site bonito e funcional..",
```

Outro ponto, isso seria a definição do que é um pré-processador, certo? Acho que se encaixaria mais do que colocar como SASS/SCSS, pois estem vários outros.

---

@NeiltonSeguins quando você diz colocar como exemplo, se refere ao código em si ou só adicionar o nome desses pré-processadores como exemplo mesmo?

---

Só adicionar os nomes como exemplo mesmo no texto.

---

Show @NeiltonSeguins! Só lembrando para olhar e atualizar o seu PR 🥹 não se esqueça de atualizar a branch também!

---

Oi, @levxyca ! Desculpa a demora em fazer essa alteração. Mas finalmente atualizei a branch e a PR conforme essa conversa. Pode revisar, por favor?